### PR TITLE
xargs: add/refresh pages

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/common/xargs.md [hunk 1]
```diff
@@ -2,24 +2,33 @@
 
 > Execute a command with piped arguments coming from another command, a file, etc.
 > The input is treated as a single block of text and split into separate pieces on spaces, tabs, newlines and end-of-file.
-> More information: <https://pubs.opengroup.org/onlinepubs/9699919799/utilities/xargs.html>.
+> See also: `find`.
```
Comment (blueskyson) [inline pages/common/xargs.md:5]:
```suggestion
```

File: pages/linux/xargs.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# xargs
+
+> Execute a command with piped arguments coming from another command, a file, etc.
+> The input is treated as a single block of text and split into separate pieces on spaces, tabs, newlines and end-of-file.
+> See also: `find`.
```
Comment (blueskyson) [inline pages/linux/xargs.md:5]:
```suggestion
```

File: pages/common/xargs.md (hunk 1 above)
Comment (EmilySeville7cfg) [reply pages/common/xargs.md:5]: There is a reference to `find` at the bottom of two pages. Why do you suggest to remove `See also: find`>

Comment (blueskyson) [reply pages/common/xargs.md:5]: The functions of `xargs` and `find` are not related.

**Review by kbdharun — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Closing PR, to make somebody pay attention to it, review and merge.

**Pull request closed without merging**